.github/workflows/pr.yml

```diff
@@ -0,0 +1,27 @@
+name: validate pull requests
+on:
+  pull_request:
+
+jobs:
+  commit:
+    runs-on: ubuntu-latest
+    # Only check commits on pull requests.
+    if: github.event_name == 'pull_request'
+    steps:
+      - name: get pr commits
+        id: 'get-pr-commits'
+        uses: tim-actions/get-pr-commits@v1.1.0
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: DCO check
+        uses: tim-actions/dco@master
```

Comment:
Non-blocker: It would be better to use a tagged version, but for such a simple thing I can understand if there isn't one.